packages/astro/components/Code.astro

```diff
@@ -0,0 +1,21 @@
+---
+import shiki from 'shiki';
+const { code, lang = 'plaintext', theme = 'nord' } = Astro.props;
```

Comment:
Can you add an interface? I think we should have those going forward. I plan on PRing a change to add interfaces for Markdown and Prism.